### PR TITLE
A new workable deployment yaml for bosh-init

### PR DESCRIPTION
**Pull request by jianqiu**

### CODE REVIEW

**Comment by maximilien** [comment]
Nice. 

@karafelix can we make sure the pipeline is indeed using this (or modified version) and fix any changes you make.

Thanks all. Cheers :beers:

File: docs/bosh-init-usage.md [hunk 1]
```diff
@@ -73,55 +81,95 @@ jobs:
...
       cpi_job: cpi
-      max_threads: 3
-
+      address: 127.0.0.1
+      name: bosh
+      db:
+        adapter: postgres
+        database: bosh
+        host: 127.0.0.1
+        password: postges
```
Comment (ardnaxelarak) [inline docs/bosh-init-usage.md:125]: I assume "postges" is a typo and should be "postgres"?

File: docs/bosh-init-usage.md [hunk 2]
```diff
@@ -20,38 +20,45 @@ name: bosh
...
   stemcell:
-    url: file://./light-bosh-stemcell-3031-softlayer-esxi-ubuntu-trusty-go_agent.tgz # <--- Stemcell and bosh-agent should need some changes due to constrains of Softlayer
+    url: file://./light-bosh-stemcell-3031-softlayer-esxi-ubuntu-trusty-go_agent.tgz
   cloud_properties:
-    Domain: softlayer.com
-    VmNamePrefix: bosh-experimental  # <--- It is better to use a catchy name which will be used in the following section
-    StartCpus: 1
-    MaxMemory: 1024
+    Domain: SOFTLAYER.COM      # <--- domain name for new director
+    VmNamePrefix: VMNAMEPREFIX # <--- indicate a hostname prefix for new director
```
Comment (ardnaxelarak) [inline docs/bosh-init-usage.md:35]: I believe Alexey's changes completely changed the 'VmNamePrefix' property to 'Hostname'.

**Comment by ardnaxelarak** [comment]
I'm waiting on bosh-init to run, but it seems to have gotten farther than I've ever gotten before, so things are looking positive. I'll submit a pull request with the changes I made (commented above) if they work.

**Pull request merged**